### PR TITLE
Seed data integration for audio files

### PR DESCRIPTION
**Pull request by DanielKuhn**
REDMINE-17547

### CODE REVIEW

File: entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake [hunk 1]
```diff
@@ -72,6 +72,14 @@ namespace :pageflow_scrolled do
...
                 }
               }.stringify_keys
+            },
+            audio_files: {
+              rocky_theme: {
+                url: 'https://s3-eu-west-1.amazonaws.com/de.codevise.pageflow.development/pageflow-next/seed-assets/audios/Rocky_Theme.m4a',
+                configuration: {
+                  testReferenceName: 'rocky_theme'
+                }
+              }.stringify_keys
```
Comment (tf) [inline entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake:82]: Do we need to think about the license of this file?

Comment (tf) [reply entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake:82]: Not yet documented here: https://github.com/codevise/pageflow/blob/0c84a00949f2c34934ebd71da5306dcc5e0beb65/entry_types/scrolled/package/spec/support/stories.js#L72-L78

**Comment by tf** [comment]
Percy failure seems to be related to inline audio, though. Can you investigate this?

**Comment by DanielKuhn** [comment]
I added #1382 as a prerequisite for this.

**Pull request merged**